plugin.xml

```diff
@@ -76,5 +76,13 @@
       <config-file target="config.xml" parent="/*">
         <access origin="*" />
       </config-file>
+
+      <!-- add required entries into plist file -->
+      <edit-config target="NSCameraUsageDescription" file="*-Info.plist" mode="merge" overwrite="true">
```

Comment:
@mark-veenstra do we have to set `overwrite` to true? iOS allows you to customize the messages so what if the developer has already set these descriptions?

Reply:
Yeah that is probably a bit aggresive to set. I removed the `overwrite`